papyrus/geo_interface.py

```diff
@@ -9,6 +9,16 @@
 from shapely.geometry import asShape
 
 
+class OperationNotAllowedError(Exception):
+    """Exception raised for operations not allowed.
+
+    Attributes:
+        message -- explanation of the error
+    """
+    def __init__(self, message):
+        self.message = message
```

Comment:
Do we really need a `message` property? Isn't the following sufficient for our use-case:

```py
class OperationNotAllowedError(Exception):
    pass
```

?